e2e_deployment_files/nestededge_middleLayerBaseDeployment_amqp.json

```diff
@@ -74,18 +71,18 @@
         },
         "mqttBroker": {
           "authorizations": [
-              {
-                  "identities": [
-                      "{{iot:identity}}"
-                  ],
-                  "allow": [
-                      {
-                          "operations": [
-                              "mqtt:connect"
-                          ]
-                      }
+            {
```

Comment:
@yophilav, @huguesBouvier, I see this deployment manifest has broker-specific policies, but does not have broker flag enabled. I had to merge some conflicts, so just wanted to bring to your attention in case I broke anything.